### PR TITLE
Hollis/dialogue manager bug

### PR DESCRIPTION
**Pull request by hollisma**
# Description

Fixed a bug where certain attributes were missing from the parameters in get_logical_form in loco_mc_agent.py (maybe introduced from a recent refactor). I got the system to stop failing by passing in parameters that fit how the parameters are used in other files, however I have no idea what the underlying objects and methods actually do. @anuragprat1k mentioned that @kavyasrinet has worked on this part so it would be great if you could look this over to make sure it's not horribly wrong. 

This PR also includes Anurag's pyro launch script changes. 

## Type of change

Please check the options that are relevant.

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] Proposes a change (non-breaking change that isn't necessarily a bug)
- [ ] Refactor
- [ ] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [ ] This is a unit test
- [ ] Documentation only change
- [ ] Datasets Release
- [ ] Models Release

## Type of requested review

Not totally sure what the differences between these are
- [x] I want a thorough review of the implementation.
- [ ] I want a high level review. 
- [ ] I want a deep design review.

## Before and After

Before: the bot responds correctly but the status shows an "Error in sending chat"
image

After: the command sends successfully
image

# Testing

Did not run any tests. 

# Checklist:

- [x] I have performed manual end-to-end testing of the feature in my environment.
- [ ] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] New and existing unit tests pass locally with my changes.
- [x] I have added relevant collaborators to review the PR before merge.

### CODE REVIEW

File: locobot/robot/launch_pyro_habitat.sh [hunk 1]
```diff
@@ -2,7 +2,14 @@
 
 export PYRO_SERIALIZER='pickle'
 export PYRO_SERIALIZERS_ACCEPTED='pickle'
+export PYRO_SOCK_REUSE=True
 
+echo "Ensuring clean slate (kills roscore, rosmaster processes)..."
```
Comment (anuragprat1k) [inline locobot/robot/launch_pyro_habitat.sh:7]: looks like this is an older commit? this isn't right change. as discussed replace the `pyro-ns` command here.

**Review by anuragprat1k — CHANGES_REQUESTED** [review]
Looks good! Requesting a minor change from what seems like a stray commit.

File: locobot/robot/launch_pyro_habitat.sh (hunk 1 above)
Comment (anuragprat1k) [reply locobot/robot/launch_pyro_habitat.sh:7]: It's odd that that command isn't showing as a diff here. You'll need to rebase your change after the most recent refactor in main - maybe that will fix it. Can you try rebasing and updating this PR? The location of the files in the new structure are [here](https://github.com/facebookresearch/droidlet/blob/main/agents/loco_mc_agent.py#L117) and [here](https://github.com/facebookresearch/droidlet/blob/main/droidlet/lowlevel/locobot/remote/launch_pyro_habitat.sh) and

File: base_agent/loco_mc_agent.py [hunk 2]
```diff
@@ -118,13 +118,13 @@ def send_text_command_to_agent(sid, command):
             logical_form = {}
             status = ""
             try:
-                logical_form = dialogue_manager.get_logical_form(
-                    s=command, model=dialogue_manager.model
+                logical_form = dialogue_manager.semantic_parsing_model_wrapper.get_logical_form(
```
Comment (kavyasrinet) [inline base_agent/loco_mc_agent.py:121]: Thanks for fixing this. This definitely looks like a merge conflict wasn't resolved properly after a recent refactor.

**Review by kavyasrinet — APPROVED** [review]

File: locobot/robot/launch_pyro_habitat.sh (hunk 1 above)
Comment (anuragprat1k) [reply locobot/robot/launch_pyro_habitat.sh]: Looks good now!

**Review by anuragprat1k — APPROVED** [review]

**Pull request merged**